src/components/utils/skills-quiz.js

```diff
@@ -0,0 +1,9 @@
+import { DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE } from '../skills-quiz/constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export function checkValidGoalAndJobSelected(goal, jobs, goalIsImprove) {
```

Comment:
nit: Rename `goalIsImprove` to `checkGoaIsImprove` or something more meaningful so the purpose of this boolean is obvious

Reply:
done